### PR TITLE
Add user mgmt docs

### PR DESCRIPTION
**Pull request by adifayer**
https://github.com/datreeio/docs/issues/75

### CODE REVIEW

File: docs/setup/user-management/index.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: User Management
+slug: /setup/user-management
+---
+
+:::info
+Feature Availability
```
Comment (eyarz) [inline docs/setup/user-management/index.mdx:7]: I think this can be the title instead of "info"

File: docs/setup/user-management/index.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+## User Roles 
+There are 3 levels of permissions, and they are defined by the user’s role:  
+- Owner
+- Admin
+- Viewer
+
+## Roles Permissions
+| Function | Owner | Admin | Viewer |
+|----------|-------|-------|--------|
+|Create/delete policies|X|X||
```
Comment (eyarz) [inline docs/setup/user-management/index.mdx:41]: Maybe change all the X to V?

File: docs/setup/user-management/index.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+:::info
+Feature Availability  
+Inviting team members to collaborate is available as part of certain payment plans. Plan details can be found in our pricing page(add link).
+:::
+
+## Invite a user
+
+### 1. In the dashboard, navigate to the [user management tab](https://app.datree.io/settings/user-management)
```
Comment (eyarz) [inline docs/setup/user-management/index.mdx:13]: why is this H3 and the rest is not?

File: docs/setup/user-management/index.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+title: User Management
```
Comment (eyarz) [inline docs/setup/user-management/index.mdx:2]: lower "m" => User management

**Comment by eyarz** [comment]
@adifayer, also please give the image a more descriptive name (static/img/setup/user-management/1.png)

**Comment by eyarz** [comment]
image

**Comment by eyarz** [comment]
@adifayer 
image

File: docs/setup/user-management/index.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+title: User management
+slug: /setup/user-management
+---
+
+:::Feature availability
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:6]: Fix tooltip, and change (add link) to an actual link;)

File: docs/setup/user-management/index.mdx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- Go to "SETTINGS" (click on the avatar in the upper-right corner)  
+![settings](/img/setup/account-token/1.png)
+- Click on the ‘invite user’ button  
+2. Insert the user’s email address and the desired permission (role)
+3. Once the invitation is successfully sent, the token of the newly invited user will be displayed in the dashboard and can be shared with the new workspace members  
+- The token is used to connect the CLI with the dashboard to determine the available policies and report policy check history to the dashboard  
+4. The newly invited user is added to the users’ table, where you can also track his/her invitation status (pending/accepted)  
+
+## Edit user’s role
+1. In the dashboard, navigate to the user management tab
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:22]: Add link here (same as you did in the "invite a user" section)

File: docs/setup/user-management/index.mdx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+## Edit user’s role
+1. In the dashboard, navigate to the user management tab  
+2. Click on the ‘...’ icon displayed for each member  
+![settings](/img/setup/user-management/edit-user.png)
+3. Click on "Edit"
+4. Select the new role and save the change  
+
+## Inivtation expired
+In cases where an invitation to join the workspace has reached the expiration date, reinvite the user by clicking on the "invite user" button and insert the email and permission.
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:29]: If an invitation to join a workspace has reached it expiration date, you can re-invite the user by clicking on the "invite user" button and providing the desired email and permission.

File: docs/setup/user-management/index.mdx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+## User Roles 
+There are 3 levels of permissions, and they are defined by the user’s role:
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:32]: There are 3 levels of permissions, defined by the user’s role:

File: docs/setup/user-management/index.mdx [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+## Inivtation expired
+In cases where an invitation to join the workspace has reached the expiration date, reinvite the user by clicking on the "invite user" button and insert the email and permission.
+
+## User Roles 
+There are 3 levels of permissions, and they are defined by the user’s role:  
+- Owner
+- Admin
+- Viewer
+
+## Roles Permissions
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:37]: Role permissions

File: docs/setup/user-management/index.mdx [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+1. In the dashboard, navigate to the user management tab  
+2. Click on the ‘...’ icon displayed for each member  
+![settings](/img/setup/user-management/edit-user.png)
+3. Click on "Edit"
+4. Select the new role and save the change  
+
+## Inivtation expired
+In cases where an invitation to join the workspace has reached the expiration date, reinvite the user by clicking on the "invite user" button and insert the email and permission.
+
+## User Roles
```
Comment (hadar-co) [inline docs/setup/user-management/index.mdx:31]: User roles

**Review by hadar-co — APPROVED** [review]

**Pull request merged**